Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -3744,6 +3744,7 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         case CSSPropertyScrollSnapAlign:
         case CSSPropertyScrollSnapStop:
         case CSSPropertyScrollSnapType:
+        case CSSPropertyScrollbarWidth:
```

Comment:
This property is [animatable according to its spec](https://w3c.github.io/csswg-drafts/css-scrollbars/#scrollbar-width). You should be able to make it animatable by adding the following line of code along where other `DiscretePropertyWrapper` objects are created:

```c+
new DiscretePropertyWrapper<ScrollbarWidth>(CSSPropertyScrollbarWidth, &RenderStyle::scrollbarWidth, &RenderStyle::setScrollbarWidth)
```

Animation support would also need to be tested, as it does not seem to be the case as it stands. You could do that by adding a new type to `LayoutTests/imported/w3c/web-platform-tests/web-animations/animation-model/animation-types/property-list.js`. Looking at that file, it should be pretty straightforward for you to add a new discrete animatable property.

Let me know if you need help.